src/codegen.cpp

```diff
@@ -6432,6 +6432,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef USE_POLLY
+    cl::ParseEnvironmentOptions("Julia", "JULIA_POLLY_ARGS");
```

Comment:
A bit of seperate control would be nice, to, for example, null the polly arguments while keeping the other LLVM args as they are.